src/components/Chat/Chat.js

```diff
@@ -84,6 +84,7 @@ Chat.propTypes = {
       title: PropTypes.string.isRequired,
     }),
     avatar: PropTypes.string,
+    errorMessage: PropTypes.node,
```

Comment:
why do we need this on top-level chat component?

Reply:
It's just for getting an overview of the props in the Storybook. It should be `error` btw.